gpkit/tools/spdata.py

```diff
@@ -26,7 +26,8 @@ def __init__(self, model):
             else:
                 raise ValueError("unknown constraint %s of type %s"
                                  % (constraint, type(constraint)))
-        NomialData.init_from_nomials(self, self.signomials)
+        # NomialData.init_from_nomials(self, self.signomials)
```

Comment:
nope, this is commented to make sure it doesn't run, because this all needs to be rewritten :-/